data/human/human missions.txt

```diff
@@ -7454,3 +7454,209 @@ mission "Adelita 5"
+			`	You note that the message appears to be coming from an emergency transmitter commonly found on many small shuttles and escape pods. There is no shortage of pirates in <waypoints>.`
+			choice
+				`	"To <waypoints>!"`
+				`	(Ignore it and walk away.)`
+					decline
+			`	You return to your ship and prepare to leave.`
+				accept
+	on accept
+		event "start of independent violence"
+	on enter "Moktar"
+		"reputation: Independent (Killable)" = -1000
+		conversation
+			`When you enter <waypoints>, your sensors detect a pirate Vanguard and a derelict escape pod aimlessly drifting near the coordinates in the original message. Your ship is immediately hailed, and the pirate opens communication.`
+			`	"You were unwise to get involved in our business. Hand over that pompous salesman Howard, and I'll let you live."`
+			choice
+				`	"Uhhh... hand over who?"`
+				`	"I don't negotiate with pirates!"`
+					goto negotiate
+				`	"You can take him, but I don't know where he is."`
+			`	"Bah, you spineless buffoon! Your lies won't save you or that useless salesman. I'll find him in your ship's wreckage!"`
+				goto power
+			label negotiate
+			`	"Aye, then I won't negotiate with you! Prepare to meet your end!"`
+			label power
+			`	The Vanguard's engines roar to life as you see over half a dozen fixed guns take aim at your ship, ready to strike.`
+	npc
+		government "Independent (Killable)"
+		personality heroic staying vindictive
+			confusion 20
+		system "Moktar"
+		ship "Vanguard" "Golden Jewel"
+		on kill
+			set "syndicate business: golden jewel dead"
+			dialog "The crew of the Golden Jewel are no more. You think you can make out the pirate captain drifting in space nearby."
+	npc save assist
+		government "Independent"
+		personality derelict target quiet escort
+		system "Moktar"
+		ship "Escape Pod" "Escape Pod 001"
+		on kill
+			dialog "You watch as the escape pod you're racing to save explodes into a cloud of debris, all hope snuffed out in an instant."
+		conversation
+			`You board the escape pod to see its desperate and nearly incoherent occupant: "Oh thank the stars, thank you! Captain, you've got to save me - please! I'm Howard. I promise you a reward if you can fix up this tin can so I can get to <destination>! I'm with Megaparsec, I can make it worth your while!"`
+			choice
+				`	"Sounds good."`
+					decline
+				`	"What's the deal with you and that pirate?"`
+			`	Howard takes a breath and seems to calm himself. "That pirate was a potential investor in the firm I work for. We had a disagreement over the terms and it obviously got quite out of hand. I'll need to find a new business partner now..." He trails off. Time to get back to <planet>.`
+	on complete
+		"reputation: Independent (Killable)" = 10
+		event "end of independent violence"
```

Comment:
```suggestion
		event "end of independent violence"
		log "Went to rescue somebody who said he was on a pirate ship. This turned out to be true, and I was offered and investment opportunity."
		log "People" "Howard" "Howard is a salesman from Megaparsec in charge of setting up investments. First met him on a pirate ship whose owner he was trying to convince to invest."
```

Reply:
Logs should avoid using first person pronouns.

Reply:
How about it now?